src/wp-admin/js/user-profile.js

```diff
@@ -374,7 +374,7 @@
 			});
 
 			/**
-			 * Replaces "Howdy, *" in the admin toolbar whenever the display name dropdown is updated for one's own profile.
+			 * Replaces "Hello, *" in the admin toolbar whenever the display name dropdown is updated for one's own profile.
```

Comment:
After the changes in #51, this should just say `the display name`.  I'll update this now.